api/v1/api.md

```diff
@@ -0,0 +1,145 @@
+# dcrtime API Specificiation
+
+V1
+
+This document describes the REST API provided by a `dcrtimed` server.  This API allows users to create upload hashes which are periodically submitted to the decred blockchain and timestamped as well as to confirm such timestamps.
+
+**Methods**
+
+- [`Timestamp`](#timestamp)
+- [`Verify`](#verify)
+
+**Return Codes**
+
+- [`ResultOK`](#ResultOK)
+- [`ResultExistsError`](#ResultExistsError)
+- [`ResultDoesntExistsError`](#ResultDoesntExistsError)
+
+### Methods
+
+#### `Timestamp`
+
+Timestampe the server to store one or more digests and return a server timestamp.
```

Comment:
Timestamp not timestampe.  The resulting timestamp is called a collection.  This result can, by default, not be used and we need to point users to use digests instead.